kyuubi-server/src/test/scala/org/apache/kyuubi/operation/thrift/http/KyuubiOperationThriftHttpKerberosAndPlainAuthSuite.scala

```diff
@@ -49,19 +51,46 @@ class KyuubiOperationThriftHttpKerberosAndPlainAuthSuite
     UserGroupInformation.setConfiguration(config)
     assert(UserGroupInformation.isSecurityEnabled)
 
-    KyuubiConf().set(KyuubiConf.AUTHENTICATION_METHOD, Seq("KERBEROS", "LDAP", "CUSTOM"))
+    KyuubiConf().set(KyuubiConf.AUTHENTICATION_METHOD, Seq("KERBEROS", "CUSTOM", "LDAP"))
```

Comment:
the UT of `KyuubiOperationKerberosAndPlainAuthSuite` will fail,

Reply:
you can refer https://github.com/apache/kyuubi/blob/0165d5e3a26cc8353a9f7135e8fa4a9c4ae99e78/kyuubi-server/src/test/scala/org/apache/kyuubi/operation/KyuubiRestAuthenticationSuite.scala#L49-L53 to adjust the UT with LDAP and CUSTOM auth